vars/parseStageInfo.groovy

```diff
@@ -70,6 +70,16 @@ String get_commit_pragma_tags(String pragma_suffix) {
   return pragma_tag
 }
 
+String get_default_nvme() {
+  // Get the default test nvme setting
+  if (env.BRANCH_NAME.startsWith('feature/vos_on_blob')) {
```

Comment:
I'm a little concerned about `startsWith()` here.  Somebody could create `feature/vos_on_blobulator` not realizing what it will mean in terms of this code.

But maybe you are thinking of a use case where `startsWith()` is useful?

Reply:
It is a temporary solution until the feature/vos_on_blob branch merges with master.  At that point we will need a different solution (possibly more stages running the same tests with different Test-nvme assignments) which will at least include removing the reference to this specific branch, but if we were to create a feature/vos_on_blob_2_6 it would be supported.

Reply:
Fair enough.